vrrpd/vrrp.c

```diff
@@ -91,7 +91,7 @@ static void vrrp_recalculate_timers(struct vrrp_router *r)
 	uint16_t skm = (r->vr->version == 3) ? r->master_adver_interval : 100;
 
 	r->skew_time = ((256 - r->vr->priority) * skm) / 256;
-	r->master_down_interval = 3 * mdiadv;
```

Comment:
![Screenshot-2024-12-12-11:26:24](https://github.com/user-attachments/assets/eb6e0139-8e14-4de9-9c6c-a90e18e39bf7)

This is defined in https://datatracker.ietf.org/doc/html/rfc5798.